platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java

```diff
@@ -0,0 +1,599 @@
+   * space that she is not member of via url : /portal/g/:spaces:open/open/forum/
+   * Click on Join link Input Data: Expected Outcome: - New space is added
+   * successfully - A page with Restricted Area title is displayed Message is :
+   * You must be a member of the space Open to access this page. [Join - Mary
+   * joins the space and is redirected to the initially requested page.
+   */
+  @Test
+  @BugInPLF("SOC-5885")
+  public void test08_CheckAccessVisibleOpenSpace() {
+    info("Test 08:Check access visible/open space");
+    String space = "space" + getRandomNumber();
+    String[] arrayRight = { "open" };
+    String mess = "You must be a member of the space " + space + " to access this page.";
+    String username1 = "usernamea" + getRandomString();
+    String email1 = username1 + "@test.com";
+    String username2 = "usernameb" + getRandomString();
+    String email2 = username2 + "@test.com";
+    String password = "123456";
+    info("Add user");
+    navigationToolbar.goToAddUser();
+    addUsers.addUser(username1, password, email1, username1, username1);
+    addUsers.addUser(username2, password, email2, username2, username2);
+    manageLogInOut.signIn(username1, password);
+    info("Create a space");
+    homePagePlatform.goToMySpaces();
+    spaceManagement.addNewSpaceSimple(space, space);
+    info("Set permission for the space");
+    spaceHomePage.goToSpaceSettingTab();
+    spaceSettingManagement.goToAccessEditTab();
+    spaceSettingManagement.setPermissionForSpace(arrayRight);
+    spaceManagement.goToActivityStreamTab();
+    refresh();
+    String urlSpace = url();
+    manageLogInOut.signIn(username2, password);
+    for (int i = 0; i <= 14; i++) {
+      open(urlSpace);
+      if ($(ELEMENT_SPACE_ACCESS_RESTRICED_AREA_TITLE).is(Condition.visible)) {
+        break;
+      }
+    }
+
+    $(ELEMENT_SPACE_ACCESS_RESTRICED_AREA_TITLE).should(Condition.visible);
+    $(ELEMENT_SPACE_ACCESS_INFO).getText().contains(mess);
+    $(ELEMENT_SPACE_ACCESS_JOIN_BTN).click();
+    manageLogInOut.signIn(DATA_USER1, "gtngtn");
+    navigationToolbar.goToManageCommunity();
+    addUsers.deleteUser(username1);
+    addUsers.deleteUser(username2);
+  }
+
+  /**
+   * <li>Case ID:121918.</li>
+   * <li>Test Case Name: Check access hidden space.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Step 1: Check hidden space
+   * Step Description: - Login as user1 - Add new hidden space - Logout - Login as
+   * user2 - access the url of an hidden space that she is not member of via url :
+   * /portal/g/:spaces:hidden/hidden/forum/ - Click on [Find Spaces] link Input
+   * Data: Expected Outcome: - Add new space successfully - A page with 'Space not
+   * Found' title is displayed Message is : No space is available at this URL.
+   * [Find Spaces] - User2 is redirected to Spaces directory page
+   */
+  @BugInPLF("SOC-5884")
```

Comment:
The issue SOC-5884 is closed as Normal Behavior. Should we remove this tag (and maybe update the test) ?